local-volume/README.md

```diff
@@ -37,11 +37,20 @@ What doesn't work and workarounds:
         * Give your pods high priority.
         * Run a workaround controller that unbinds PVCs for pods that are
           stuck pending. TODO: add link
-* External provisioner cannot correctly detect capacity of mounts added after it
-  has been started.
-    * This requires mount propagation to work, which is targeted for 1.9.
-    * Workaround: Before adding any new mount points, stop the daemonset, add
-      the new mount points, start the daemonset.
+* The provisioner does not correctly detect mounts added after it
+  has been started due to lack of mount propogation.
+  * The provisioner may also incorrectly create local PVs with the root
```

Comment:
I would keep the first sentence and  s/may/will.

The rest of this paragraph I would remove because the race condition doesn't exist in current code.